frontend/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactChild } from 'react'
+import { IconType } from 'react-icons/lib'
+import {
+  Box,
+  forwardRef,
+  HStack,
+  Icon,
+  Text,
+  useMultiStyleConfig,
+  VStack,
+} from '@chakra-ui/react'
+
+export interface TileProps {
+  // The typing here is to satisfy the ts compiler
+  // because otherwise, it will complain about assigning null to the as prop
+  /**
+   * The icon that should be displayed on the tile
+   */
+  icon: IconType
+
+  /**
+   * The title of the tile
+   */
+  title: string
+  /**
+   * The subtitle of the tile
+   */
+  subtitle: string
+
+  /**
+   * The components to be displayed below the title/subtitle
+   */
+  children: ReactChild
+
+  /**
+   * The tag, if any, to be displayed alongside the title
+   */
+  tag?: JSX.Element
+}
+
+type TileVariant = 'complex' | 'simple'
+export const Tile = forwardRef<TileProps, 'div'>(
+  ({ tag, icon, title, subtitle, children }, ref) => {
+    const variant: TileVariant = children ? 'complex' : 'simple'
+    const styles = useMultiStyleConfig('Tile', { variant })
+
+    return (
+      // Ref passed into the component as a whole so that it can be focused
+      <Box sx={styles.container} ref={ref}>
+        <VStack spacing="1rem" alignItems="flex-start">
```

Comment:
Should we be putting all styles into the theme?

Reply:
i agree with putting _parts_ of the component into the theme; however, i think this is more of a utility component (thus, not a part) and should be left as is, cos otherwise our themes run the risk of being overloaded @__@

i'll experiment with using `VStack` as the outer component and removing teh box so that we can put it in themes!

Reply:
Can always use the `_first` selector to only apply the styling to the first child